### PR TITLE
imager: update to v3.9.11 (jan24)

### PR DESCRIPTION
**Pull request by PenguenJulien**
###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by PenguenJulien** [comment]
Hello,

I don't understand the error : 

Error: reinplace: couldn't read file "/opt/local/var/macports/build/_Users_runner_work_macports-ports_macports-ports_ports_science_imager/imager/work/imager-3.9-11/admin/define-system.sh": no such file or directory

But the file exist ...

Could you help me please?

Thanks

File: science/imager/Portfile [hunk 1]
```diff
@@ -24,22 +24,16 @@ homepage            https://imager.oasu.u-bordeaux.fr
...
+worksrcdir          ${name}-${source_version}
 
-checksums           rmd160  0279d3cea7d966433eeda9909d2abc26db483eb3 \
-                    sha256  4e29dd0d740e6549934116a0bd34834950fe3179fb3652e30ea888ac049fc67c \
-                    size    32453347
+checksums           rmd160  31b94aa271f4835f3ab8583453ea6fba2702aa51 \
+                    sha256  5524f72afc9db8151d652e0994e2ee5db75fb7cb6a8bc91cd59676ef24ebcb6f \
+                    size    32484632
 
 set py_ver          3.11
```
Comment (reneeotten) [inline science/imager/Portfile:34]: this should use Python 3.12 now if possible.

**Comment by pmetzger** [comment]
@PenguenJulien @penguen Do you have two accounts on Github?

**Comment by PenguenJulien** [comment]
hi,
no only one account on Github : @PenguenJulien

**Comment by reneeotten** [comment]
> hi,
> 
> no only one account on Github : @PenguenJulien 

so the current GitHub handle for the maintainer is not you?

**Comment by PenguenJulien** [comment]
don't use @penguen it's not me, use @PenguenJulien

**Comment by reneeotten** [comment]
okay, so you are not the current maintainer of this port then...?

**Comment by PenguenJulien** [comment]
yes i am the current maintainer, see https://github.com/PenguenJulien (see on my repository)
I don't understand your question, i use only one account on Github

**Comment by reneeotten** [comment]
then look at what GitHub handle you have listed in the maintainer field. That's where the confusion comes from. So if that is not you, then you either added the wrong GitHub handle at some point or you are actually not the maintainer of the port... That needs to be clarified.

**Comment by PenguenJulien** [comment]
When i ass for pull request macportsbot Notifying maintainers @penguen for port imager but it's a mistake, i don't understand why ... my only account it's @PenguenJulien and i am the maintainer, could you change that?

**Comment by reneeotten** [comment]
You added that GitHub handle yourself when the port was initially submitted (990f138); so if that was a mistake then please correct the maintainer line as part of this PR. Once you have made the corrections, please squash all the commits into one so that this can be merged.

**Comment by pmetzger** [comment]
@PenguenJulien The `maintainers` line, which you created, says:

```
maintainers         {u-bordeaux.fr:julien.penguen @penguen}
```

**Comment by reneeotten** [comment]
I am not going to engage any further here in this PR. I am not sure what I else I can say to make things more clear...

**Pull request closed without merging**